test/helper/sending.bash

```diff
@@ -56,11 +56,12 @@ function _send_mail_and_get_id() {
 
   MAIL_ID=$(_exec_in_container tac /var/log/mail.log              \
     | grep -E -m 1 'postfix/smtpd.*: [A-Z0-9]+: client=localhost' \
-    | grep -E -o '[A-Z0-9]{11}')
+    | grep -E -o '[A-Z0-9]{9,12}' || true)
```

Comment:
Forgot to mention this (FYI): On aarch64, the ID postfix uses might not be 11 characters long, so I provide a bit of range here.

Reply:
I don't have the slightest idea.. but this is what I saw today on my arm64 machine.